rules/windows/defense_evasion_code_signing_policy_modification_registry.toml

```diff
@@ -99,7 +99,7 @@ registry where host.os.type == "windows" and event.type : ("creation", "change")
 (
   registry.path : "HKEY_USERS\\*\\Software\\Policies\\Microsoft\\Windows NT\\Driver Signing\\BehaviorOnFailedVerify" and
   registry.value: "BehaviorOnFailedVerify" and
-  registry.data.strings : ("0", "0x00000000", "1", "0x00000001")
+  registry.data.strings : ("0", "0x00000000")
```

Comment:
Would it make sense to reflect this change in the rule name as well?

Reply:
I think we can leave it as is

Reply:
@Aegrah I was taking a second look at this one, and it seems that the original logic is the way to go here

![image](https://github.com/elastic/detection-rules/assets/26856693/8e8f33e1-4c56-436e-a085-f2abcf270097)

Will exclude it for now, and explore other tuning opportunities